mex/editor/aux_search/main.py

```diff
@@ -172,16 +175,51 @@ def search_results() -> rx.Component:
 def nav_bar() -> rx.Component:
     """Render a bar with an extractor navigation menu."""
     return rx.flex(
-        rx.foreach(
-            AuxState.aux_data_sources,
-            lambda item: rx.text(
-                item,
-                cursor="pointer",
-                size="5",
+        rx.tabs.root(
+            rx.tabs.list(
+                rx.spacer(),
+                rx.tabs.trigger("Wikidata", value="wikidata"),
+                rx.tabs.trigger("LDAP", value="ldap"),
+                rx.tabs.trigger("Orchid", value="orchid"),
```

Comment:
could you move this into an attribute on the state? similar to https://github.com/robert-koch-institut/mex-editor/blob/0.12.0/mex/editor/state.py#L17

knowing about which "aux providers" we have and which one the default is, should be the job of the business layer, not the presentation layer[^1]. 

[^1]: https://en.wikipedia.org/wiki/Multitier_architecture